master_history.txt

```diff
@@ -7,4 +7,10 @@ tag: 2.7.24     date: 11/05/2018  master: 1a7b524b    develop: fab89e37
 tag: 2.8.00     date: 02/05/2019  master: a6e05e06    develop: 6a790321
 tag: 2.9.00     date: 06/24/2019  master: 4ee5f3c6    develop: 094da30c
 tag: 3.0.00     date: 01/31/2020  master: d86db111    release-candidate-3.0: cf24ab90
-tag: 3.1.00     date: 04/14/2020  master: f199f45d    develop: 8d063eae 
+tag: 3.1.00     date: 04/14/2020  master: f199f45d    develop: 8d063eae
+tag: 3.1.01     date: 05/04/2020  master: 43773523    release: 6fce7502 
```

Comment:
Something is off, why is this file changed?

Reply:
Sorry I shouldn't have committed either of these (master_history or CHANGELOG). It was an artifact of merging/reflogging my slightly outdated branch into the current develop. I'll revert these to the current 3.5 ones.